magento/magento2
# Allow meta descriptions to be longer.

Google has increased the maximum allowed length for meta descriptions to 320 characters. Due to this attribute being a varchar, we are forced to change it into a 'text' type to avoid a 255 length limitation.

### Description
Steps that are taken when upgrading data:
- Change backend_type of attribute to text + note to reflect change
- Select all attribute values from `catalog_product_entity_varchar` and insert them into `catalog_product_entity_text`.
- Remove attribute values from `catalog_product_entity_varchar`.

### Manual testing scenarios
Before upgrading to this version, make sure you have various products with meta descriptions to check that data gets migrated correctly.

### Contribution checklist
